### ISSUE TITLE
the argument of C&W attack API

### ISSUE THREAD

**Issue by sndnyang** [P2333/Max-Mahalanobis-Training#2]
Hi, I have a question in the advtest_other.py

row 283:  adv_x = att.generate(x_place, is_MMC=is_MMC, **att_params)

But the att is from the cleverhans library. the official API doesn't have this argument. So I think you modify the source code of cleverhans a bit.

Can you upload the modified source file? I hope it's just one file.

**Comment by P2333** [comment]
Our evaluation is based on cleverhans: 2.1.0. To perform adaptive attack, please manually modify the function `model_loss` of the file `utils_tf.py` (in the source code of cleverhans) by substituting the softmax cross-entropy loss with other adaptive objectives, e.g., `out=-tf.reduce_sum(logits * y, axis=-1)`. The API of the `att` function can simply add an argument to activate the adaptive loss
